cristin-commons/src/main/java/no/unit/nva/cristin/model/CristinOrganization.java

```diff
@@ -67,17 +67,17 @@ public boolean equals(Object o) {
         if (this == o) {
             return true;
         }
-        if (!(o instanceof CristinOrganization)) {
+        if (o == null || getClass() != o.getClass()) {
             return false;
         }
-        CristinOrganization that = (CristinOrganization) o;
-        return Objects.equals(getInstitution(), that.getInstitution()) && Objects.equals(
-            getInstitutionUnit(), that.getInstitutionUnit());
+        var that = (CristinOrganization) o;
+        return Objects.equal(institution, that.institution)
+               && Objects.equal(institutionUnit, that.institutionUnit);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(getInstitution(), getInstitutionUnit());
+        return Objects.hashCode(institution, institutionUnit);
```

Comment:
why are you changing this?
See also: https://www.baeldung.com/java-objects-hash-vs-objects-hashcode

Reply:
Sorry sorry, my bad (and intellij), had no intentions to change it

Thanks for the link.